go.mod

```diff
@@ -255,6 +255,7 @@ replace (
 	github.com/go-logr/zapr => github.com/go-logr/zapr v1.2.0
 	github.com/go-openapi/jsonpointer => github.com/go-openapi/jsonpointer v0.19.5
 	github.com/go-openapi/jsonreference => github.com/go-openapi/jsonreference v0.19.5
+	github.com/go-openapi/spec => github.com/go-openapi/spec v0.20.6
```

Comment:
this is unexpected... make sure you don't have stale generated openapi files (run `make clean generated_files`  before running `hack/update-vendor.sh`)